src/compiler/scala/tools/nsc/settings/AbsScalaSettings.scala

```diff
@@ -33,7 +33,7 @@ trait AbsScalaSettings {
   def ChoiceSetting(name: String, helpArg: String, descr: String, choices: List[String], default: String): ChoiceSetting
   def ChoiceSettingForcedDefault(name: String, helpArg: String, descr: String, choices: List[String], default: String): ChoiceSetting
   def IntSetting(name: String, descr: String, default: Int, range: Option[(Int, Int)], parser: String => Option[Int]): IntSetting
-  def MultiStringSetting(name: String, helpArg: String, descr: String): MultiStringSetting
+  def MultiStringSetting(name: String, helpArg: String, descr: String, default: List[String] = Nil): MultiStringSetting
```

Comment:
This doesn't look right to me. Compare `MultiChoiceSetting` where default arg is on MutableSettings.